src/ubuntu/18.04/amd64/Dockerfile

```diff
@@ -5,20 +5,20 @@ FROM ubuntu:18.04
 # them in a different layer.
 RUN apt-get update \
```

Comment:
Ubuntu 18.04 is end of life for almost a year. You should switch to a newer Ubuntu version that is still supported. Otherwise, you will get a security compliance issue filled against you sooner or later.

Reply:
I think it would be best to delete whole Ubuntu 18.04 directory here so that we do not have to maintain and build these docker files.

Reply:
Actually, ignore this. I got confused. Ubuntu 18.04 flavors are end of life, but Ubuntu 18.04 LTS is still supported.

Reply:
ok. We have other places where we use ubuntu 18. let me do that in separate steps. 
Unless there is strong objection @jkotas I would merge and try CI to verify that the crash is gone (it was for me running local build) 
I will follow-up with 18 upgrades to 20 or 22. That may take more testing.

BTW do you know if we plan to drop support for Ubuntu18 in 7.0? We seems to claim support even for Ubuntu 16: https://github.com/dotnet/core/blob/main/release-notes/6.0/supported-os.md

Reply:
As I have said, ignore my above comment. Ubuntu 18.04 is still supported. I have looked at a wrong page.